package.json

```diff
@@ -2,15 +2,12 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "jshint src && jscs src && pulp build --censor-lib --strict",
-    "test": "pulp test"
+    "build": "pulp build -- --censor-lib --strict"
   },
   "devDependencies": {
-    "jscs": "^3.0.7",
-    "jshint": "^2.9.4",
```

Comment:
Could include the eslint thing we have for other FFI-based libraries in here, to replace jshint/jscs?